src/components/Nav.js

```diff
@@ -1,4 +1,6 @@
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 function Nav() {
   return (
```

Comment:
- [ ] The `bookstore cms ` logo ought to be inline and not one below the other. Kindly fix this.
- [ ] The book and categories links in the nav ought to use the `Montserrat` font family. Kindly also reduce the size of the fonts to match the given design.
- [ ] The user icon positioned at the very end of the navbar is missing.

Reply:
The icon is available, you may be missing something, with npm i, everything should be available

![icon](https://user-images.githubusercontent.com/53124548/132092567-3b50f58b-db97-4382-a882-73e48821e0ff.png)

Please let me know what you mean here "The bookstore cms logo ought to be inline and not one below the other. Kindly fix this.". These are for the icon from for font awesome. Check the screenshot about let me know what you mean